doc/ug/conf.py

```diff
@@ -46,10 +46,9 @@
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 
-extensions = ['sphinx.ext.pngmath',
-              'matplotlib.sphinxext.only_directives',
-              'matplotlib.sphinxext.plot_directive',
+extensions = ['sphinx.ext.mathjax',
               'sphinx.ext.autodoc',
+              'sphinx.ext.napoleon',
```

Comment:
Is this really a good idea? I know the Google Python docstring style is much nicer than the standard sphinx rst one, but for the moment all our docstrings are written in the standard sphinx rst one anyway, and this is yet another dependency that the user may not have (doesn't seem to be on MPIP desktops, for instance)